qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -319,5 +316,39 @@ private static void createTestSamFile( ) {
 					Level.WARNING, "IOException caught whilst attempting to write to SAM test file: " + input.getAbsolutePath(), e);
 		}  
 	}
-	
+		 
+	@Test
+	public void maxLengthTest() throws Exception {	
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception